tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -291,7 +291,7 @@ def add_dim3(kernel_string, cuda_kernel):
         elif c == ")":
             closure -= 1
         elif (c == "," or ind == len(kernel_string) - 1) and closure == 0:
-            arg_locs[count]['end'] = ind
+            arg_locs[count]['end'] = ind + (c != ",")
```

Comment:
You'll want to mention in the commit message that you fixed this, and what exactly you fixed.